### PR TITLE
removes some punctuation from statue names

### PR DESCRIPTION
**Pull request by Ktlwjec1**
ok

### CODE REVIEW

**Comment by patpol4** [comment]
speed

File: code/game/objects/structures/statues.dm [hunk 1]
```diff
@@ -66,7 +66,7 @@
 	icon_state = "nuke"
 
 /obj/structure/statue/uranium/eng
-	name = "Statue of an engineer"
+	name = "statue of an engineer"
```
Comment (Bibby0110) [inline code/game/objects/structures/statues.dm:69]: not punctuation pls remove

Comment (Ktlwjec1) [reply code/game/objects/structures/statues.dm:69]: no

Comment (Bibby0110) [reply code/game/objects/structures/statues.dm:69]: >:(

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**